doozer/doozerlib/cli/scan_osh.py

```diff
@@ -579,6 +580,18 @@ def create_update_jira_ticket(self, component):
                 # Looks like it's the same NVR
                 return
 
+            if kind == BuildType.RPM:
+                # Get the build in the currently "open" ticket
+                _, build_nvr_on_ticket = self.get_scan_details_from_ticket(description=issue.fields.description)
+                build_on_ticket = self.koji_session.getBuild(build_nvr_on_ticket)
+
+                if build_on_ticket["build_id"] > build["build_id"]:
```

Comment:
```suggestion
                if int(build_on_ticket["build_id"]) > int(build["build_id"]):
```
small suggestion

Reply:
When I checked the API ,it returned int. But makes sense for us to enforce. Updated in https://github.com/openshift-eng/art-tools/pull/297/commits/dd02866f7b3a87861b285c8b57652c2168ccc4f0